src/app/views/query-runner/query-input/auto-complete/AutoComplete.tsx

```diff
@@ -57,13 +60,34 @@ class AutoComplete extends Component<IAutoCompleteProps, IAutoCompleteState> {
     this.props.contentChanged(userInput);
   };
 
+  public isOverflowing = (input: string) => {
+
+    function getTextWidth(text: string) {
+      const canvas = document.createElement('canvas');
+      const context = canvas.getContext('2d');
+    
+      if (context === null) {
+        return 0;
+      }
+
+      context.font = getComputedStyle(document.body).font;
+      return context.measureText(text).width + 5;
+    }
+
+    return this.element !== null
```

Comment:
Instead of checking for null and undefined, its possible in JavaScript to check if the variable is truthy using the double exclamation marks `!!this.element`

https://brianflove.com/2014-09-02/whats-the-double-exclamation-mark-for-in-javascript/